velox/connectors/hive/SplitReader.h

```diff
@@ -93,7 +94,7 @@ class SplitReader {
       std::shared_ptr<common::MetadataFilter> metadataFilter,
       dwio::common::RuntimeStatistics& runtimeStats);
 
-  virtual uint64_t next(int64_t size, VectorPtr& output);
+  virtual uint64_t next(uint64_t size, VectorPtr& output);
```

Comment:
> Leave it as signed for safer comparison

@Yuhta The `HiveDataSource::next(uint64_t size, velox::ContinueFuture& /*future*/)` and `Reader::next(next(uint64_t size,velox::VectorPtr& result,..)` both use uint64_t, and size never gets changed. It doesn't seem useful to make it int64_t. Shall we make all the 3 function signatures align?

Reply:
OK let's use `uint64_t` to keep it consistent with data source and file readers in this case then.  In general for new interfaces we should prefer signed types for any quantity variable, because it's much safer with the combination of subtraction and comparison